packages/core/core/src/resolveOptions.js

```diff
@@ -120,6 +120,7 @@ export default async function resolveOptions(
     outputFS,
     cache,
     packageManager,
+    reporters: initialOptions.reporters ?? [],
```

Comment:
Would personally prefer this to be more explicit like additionalReporters, not sure what other ppl think about this

Reply:
Should reporters be in .parcelrc at all or should they just be an API/CLI thing?

Reply:
That would make sense for all of the current official reporters, they are all either CLI related or only used in development.
But there are also cases like https://www.npmjs.com/package/@kicker-matchday/parcel-reporter-bundle-manifest (where you'd still want to run them every time). 

So one possibility would still be allowing reporters in the config but we could pass the cli and dev-server reporter from the CLI.

Reply:
Yeah I like that. Then we could also only add the dev server reporter in parcel serve.

Reply:
The problem with that is though: where are the reporters resolved from? We'd have to always autoinstall them

Reply:
Yeah. How do we do it for the defaultConfig option. I guess it should match?

Reply:
`defaultConfig` is just a filepath to the JSON file. Where would we get the reporter package name from? Reading the name in package.json relative to the filepath?